From 3b8485a75c5fd9a7f7877961ceadbc6521dcf563 Mon Sep 17 00:00:00 2001
From: Yuta Saito <kateinoigakukun@gmail.com>
Date: Thu, 8 Aug 2024 17:09:49 +0000
Subject: [PATCH] [CMake] Remove POSITION_INDEPENDENT_CODE=FALSE from
 BlocksRuntime

We ported BlocksRuntime CMakeLists.txt from the state of
02b7d8f0c141b9accdade1922a080898d2d0e0a2 but we don't find any
reason to set POSITION_INDEPENDENT_CODE=FALSE for BlocksRuntime.
---
 Sources/CoreFoundation/BlockRuntime/CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Sources/CoreFoundation/BlockRuntime/CMakeLists.txt b/Sources/CoreFoundation/BlockRuntime/CMakeLists.txt
index fe5e13bb..47779784 100644
--- a/Sources/CoreFoundation/BlockRuntime/CMakeLists.txt
+++ b/Sources/CoreFoundation/BlockRuntime/CMakeLists.txt
@@ -26,7 +26,4 @@ target_include_directories(BlocksRuntime PUBLIC
     # For CFTargetConditionals.h
     ${CMAKE_CURRENT_SOURCE_DIR}/../include)
 
-set_target_properties(BlocksRuntime PROPERTIES
-    POSITION_INDEPENDENT_CODE FALSE)
-
 set_property(GLOBAL APPEND PROPERTY Foundation_EXPORTS BlocksRuntime)
-- 
2.43.2

